### PR TITLE
Docs: Point to styling-webpack addon in styling docs

### PR DESCRIPTION
**Pull request by ShaunEvening**
## What I did

- Fix snippets for addon-themes docs
- Point users to addon-styling-webpack on styling and css page
- Change snippets of addon-styling to use addon-styling-webpack

## Checklist for Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### CODE REVIEW

**Review by jonniebigodes — APPROVED** [review]
@Integrayshaun thanks for taking the time to make the adjustments. Appreciate it 🙏 ! All is good on my end. Merge away when able.

File: docs/configure/styling-and-css.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 'Styling and CSS'
 ---
 
-There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration. To make this easier, we recommend using [`@storybook/addon-styling`](https://github.com/storybookjs/addon-styling).
+There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration. To make this easier, we recommend using [`@storybook/addon-styling-webpack`](https://storybook.js.org/addons/@storybook/addon-styling-webpack/).
```
Comment (kylegach) [inline docs/configure/styling-and-css.md:5]: Worth calling out the type of projects this should (or should not) be used with?

Comment (Integrayshaun) [reply docs/configure/styling-and-css.md]: Great call out! I've added a few lines about this.

**Review by kylegach — APPROVED** [review]
Nice!

**Pull request merged**